docs/developer/quickstart.md

```diff
@@ -84,17 +84,39 @@ This will build the docker image `nginx-kubernetes-gateway:<your-user>`.
    kind load docker-image nginx-kubernetes-gateway:$(whoami)
    ```
 
-3. Modify the image name and image pull policy for the `nginx-gateway` container in the
-   NKG [deployment manifest](/deploy/manifests/deployment.yaml). Set the image name to the image you built in
-   the previous step and the image pull policy to `IfNotPresent`, so that Kubernetes will not try to pull it from
-   the DockerHub. Once the changes are made, follow
-   the [installation instructions](/docs/installation.md) to install NKG on your `kind` cluster.
+3. You can use [Helm](/deploy/helm-chart/README.md) to install NKG on your `kind` cluster. Make sure to set the image
```

Comment:
I think it would be great to keep this doc as simple as possible. What do you think about simplifying steps 3 and 4 to:

3. Install Gateway API Resources
  ```
  kubectl apply -f ...
  ```
4. Install NKG using your custom image and expose NKG with a NodePort Service:
  - To install with helm:
    ```
    helm install ...
    ```
  - To install with manifests:
    ```
    make generate-manifests ...
    kubectl apply -f deploy/manifests/nginx-gateway.yaml
    kubectl apply -f deploy/manifests/nodeport.yaml
    ```
   <Maybe explain what generate-manifests does, but I don't think it's strictly necessary>

 For more information on helm configuration options see the helm [readme](link).

Reply:
I really like this, thanks!!